storage/wdpost/wdpost_run.go

```diff
@@ -426,6 +427,14 @@ func (s *WindowPoStScheduler) runPoStCycle(ctx context.Context, manual bool, di
 				return nil, xerrors.Errorf("failed to get window post type: %w", err)
 			}
 
+			if nv == network.Version19 && os.Getenv("LOTUS_NV19_OLD_PROOFS") == "1" {
```

Comment:
Note that I'm NOT switching to old proofs in the `checkProvable` step, because that is unnecessary.